alternator/executor.cc

```diff
@@ -2495,6 +2536,9 @@ std::optional<rjson::value> executor::describe_single_item(schema_ptr schema,
         // called describe_multi_item(), not this function.
         throw std::logic_error("describe_single_item() asked to describe multiple items");
     }
+    if (!consumed_capacity_collector.is_dummy()) {
+        consumed_capacity_collector += get_result_row_length(selection, *result_set->rows().begin());
```

Comment:
I think internally (initially?) we'd want to save the lengths in a histogram, so we can have the distribution + peak values.